tests/unit/customizations/test_paginate.py

```diff
@@ -19,7 +23,6 @@
 from awscli.customizations import paginate
 from awscli.customizations.exceptions import ParamValidationError
 
-
```

Comment:
nit: we should keep this line for formatting reasons, two new lines after the imports.